src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp

```diff
@@ -1498,7 +1534,10 @@ class npc_valkyr_shadowguard : public CreatureScript
                 {
                     case POINT_DROP_PLAYER:
                         DoCastAOE(SPELL_EJECT_ALL_PASSENGERS);
-                        me->DespawnOrUnsummon(1000);
+                        if (IsHeroic())
+                            me->GetMotionMaster()->MoveTargetedHome();
```

Comment:
AFAIK they move to the middle of the platform, not home pos. Home pos can be anywhere on the platform, depends where group tanks lich king